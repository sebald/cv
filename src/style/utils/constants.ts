export const htmlFontSize = 16;

export const ratio = {
  perfectFourth: 1.333,
  perfectFifth: 1.5,
  goldenRatio: 1.618,
  majorSixth: 1.667,
};
