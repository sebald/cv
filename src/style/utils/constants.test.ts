import { htmlFontSize, ratio } from './constants';

test('base font size', () => {
  expect(htmlFontSize).toMatchSnapshot();
});

test('available ratios', () => {
  expect(ratio).toMatchSnapshot();
});
