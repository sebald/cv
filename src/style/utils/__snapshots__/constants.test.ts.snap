// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`available ratios 1`] = `
Object {
  "goldenRatio": 1.618,
  "majorSixth": 1.667,
  "perfectFifth": 1.5,
  "perfectFourth": 1.333,
}
`;

exports[`base font size 1`] = `16`;
